### DISCUSSION TITLE
【2023.4.6】community meeting notes

### DISCUSSION THREAD

**Discussion by Stevent-fei** [sealerio/sealer#2171] [General]
hi all, 今晚8点，有咱们sealer的开发者会议，大家有什么议题可以提前准备哦。

**Comment by kakaZhou719** [comment]
https://github.com/sealerio/sealer/issues/2172

**Comment by starnop** [comment]
add sealer image repo and do not provide the office sealer cluster image

**Comment by kakaZhou719** [comment]
4月6日 【会议纪要】


1. sealer 0.10.0 版本计划和进度同步。@kakaZhou719
2. 本地化运行sealer image 和E2E test，需要arm机器测试，以及对应的arm infra镜像。@zhy76
3. k0s代码优化，预计5月下旬完成。 @starComingup
4. sealer app 功能增强issue 同步 @kakaZhou719
5. sealer 支持oci registry，需要测试环境，目前还有3个task待完成。@jsparter
6. sealer 不在维护官网k8s镜像，只提供basefs和container runtime 镜像。@Starnop
7. kyverno 支持动态维护policy规则，用controller 方案待定。@YTGhost
